otter/convergence/steps.py

```diff
@@ -70,12 +102,38 @@ def got_name(random_name):
                 'POST',
                 'servers',
                 data=thaw(server_config),
-                success_pred=has_code(202))
+                success_pred=has_code(202),
+                reauth_codes=(401,))
 
         def report_success(result):
             return StepResult.SUCCESS, []
 
         def report_failure(result):
+            """
```

Comment:
I wouldn't consider it a blocker but would prefer it get done here as this code does more of error parsing than _reporting_. The similar code to parse is also there in [`otter.utils.http.UpstreamError`](https://github.com/rackerlabs/otter/blob/master/otter/util/http.py#L59). I just want to ensure that code is not duplicated. I don't feel particularly strong about it as long as its done eventually. I leave it to @cyli to decide whether to do it here or not :)